eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/PageNavItem.js

```diff
@@ -25,10 +25,21 @@ const getIcon = pageType => {
   return PageIcon;
 };
 
+const getBadgeText = (page, canShowValidation) => {
```

Comment:
This change is not required for this. You can remove it. We have a separate story that will probably use websockets to get this information for all future updates. So leave it off for the moment otherwise we are going to have to maintain it as we add more validations.

Reply:
Navigation badge now removed